dm/ctl/common/util.go

```diff
@@ -33,13 +36,75 @@ import (
 	toolutils "github.com/pingcap/tidb-tools/pkg/utils"
 	"github.com/spf13/cobra"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
 var (
-	masterClient pb.MasterClient
 	globalConfig = &Config{}
+	ctlClient    = &CtlClient{}
 )
 
+// CtlClient used to get master client for dmctl
+type CtlClient struct {
+	tls          *toolutils.TLS
+	etcdClient   *clientv3.Client
+	conn         *grpc.ClientConn
+	masterClient pb.MasterClient
+}
+
+func (c *CtlClient) updateMasterClient() error {
```

Comment:
this function maybe concurrently called by user's command, so `c.conn` and `c.masterClient` may changed unexpected